ntbs-service/deployments/int.yml

```diff
@@ -85,8 +85,6 @@ spec:
             value: "development"
           - name: EnvironmentDescription__EnvironmentName
             value: "azure-int"
-      imagePullSecrets:
-      - name: registry-password
```

Comment:
Because the github registry is public, we don't need the image pull secret anymore